security/default-backend/ndn-lite-default-ecc-impl.c

```diff
@@ -16,6 +16,21 @@
 #include "../../ndn-enums.h"
 #include <string.h>
 
+static uint64_t g_rand = 88172645463325252ull;
```

Comment:
RNG belongs to RNG backend.
Pseudo RNG compromises security and must not be used.
Without RNG, ECC backend should refuse to generate keys (i.e. return error code), and select (at compile time) deterministic signing algorithm.